### DISCUSSION TITLE
Conflict between gps and lipo

### DISCUSSION THREAD

**Discussion by 19MarcoM85** [iNavFlight/inav#7567] [Q&A] [answered]
Hi everyone, I have a problem with the gps. I have a matek with compass and when I connect the fc via usb the satellites are recognized, while if I connect the lipo there is no fix. I have the same problem with both INAV and betaflight. The gps is correctly connected to the uart6 as described in the manual (the fc is a flywoo goku F722 aio) and the problem is not solved even by changing the uart and power pad. When connected via usb, only the red led of the power lights up, while connecting the lipo sometimes only the red led remains lit, sometimes the blue led of the satellite connection also lights up, but it flashes 4 - 5 times and then goes off .

I've tried everything but can't find a solution.
Help!!! : D

**Comment by DzikuVx** [comment, marked as answer]
This is not a conflict. The reason is VTX that is powered on together with everything else and pollutes RF spectrum. See here https://youtu.be/PQZmyayIrp8

**Reply by 19MarcoM85** [reply]
Thank you for reply! So the only solution Is to buy a Better GPS? Ok i have a matek M8Q-5883 and the problem remain 😅 I noticed that when i disconnect the cam (caddx tarsier) magically the gps works! Disconnecting only the vtx instead does not work. How do I prevent the tarsier from affecting the GPS?

**Comment by MrD-RC** [comment]
The Tarsier is known to cause issues with GPS. Move the GPS as far away from the Tarsier as you can.

**Reply by 19MarcoM85** [reply]
Mmm ok but it Is a 4", a Crocodile baby and i think there isnt a Place farther than a posterior arm 🤔
